### ISSUE TITLE
Follow notifications don't have focus ring (inconsistency)

### ISSUE THREAD

**Issue by MightyPork** [tootsuite/mastodon#4482]
Relating to #4448 and #4474, follow notifications lack focus ring / outline (and also don't have tabindex)

Time for 1.5.1 !!1

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by sorin-davidoi** [comment]
I think we need to consider merging the notifications component. It has a lot of overlap with StatusList but still a separate component - this is why it did not receive the accessibility fixes.

**Comment by MightyPork** [comment]
There was actually a project to refactor and DRY all the timelines at glitch-soc, but it was never merged for fear of conflicts with upstream - https://github.com/glitch-soc/mastodon/pull/27

**Comment by sorin-davidoi** [comment]
Now that I think of it doesn't even have the IntersectionObserver optimizations...

**Comment by beatrix-bitrot** [comment]
@sorin-davidoi so... will you DRY it up ?!

:3c

**Comment by sorin-davidoi** [comment]
Will see what I can do :thinking:

**Comment by ykzts** [comment]
I think it was fixed in #4697.
